src/main/java/com/prgms/allen/dining/domain/restaurant/entity/Restaurant.java

```diff
@@ -156,6 +156,10 @@ public List<Menu> getMinorMenu() {
 		return this.menu.subList(0, 4);
 	}
 
+	public boolean isAvailable(int totalCount, int requestCount) {
```

Comment:
a: 동작을 조금만 더 세세하게 명시해주는건 어떤가요?

```suggestion
	public boolean isAvalibleToReserve(int totalCount, int requestCount) {
```